### ISSUE TITLE
High-contrast themes

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#1646]
The default theme already meets WCAG accessibility guidelines AFAIK, but a high-contrast mode may help people who need even more contrast.

I'm not an expert in this area, but I suppose what would be nice is to at least have a dark and light high-contrast theme. Potentially we can hook into the Microsoft-specific high-contrast mode as well (or the standard): https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/Accessibility/HighContrast/explainer.md

**Comment by NickColley** [comment]
They don't meet WCAG at the moment, I'm going to pick this up and work on a proposal this weekend.

**Comment by NickColley** [comment]
Some feedback from a user that might benefit from more high contrast theme support:

https://dragonscave.space/@BlindMoon38/109537857318436397

Small steps might be to make some more adjustments to pitchblack too...
